source/admin_manual/list_of_events.rst

```diff
@@ -2832,18 +2834,24 @@ in to the backend.
 +=====================+======================================================+
 | source_port         | Source port where proxy connection originated from   |
 +---------------------+------------------------------------------------------+
+| reconnect_attempts  | Number of times connection failed and reconnection   |
+|                     | was attempted.                                       |
++---------------------+------------------------------------------------------+
 
 proxy_session_finished
 ----------------------
 Emitted when proxying has ended. Either successfully or with error.
 
-+-----------------+------------------------------------------------------+
-| Field           | Description                                          |
-+=================+======================================================+
-| source_port     | Source port where proxy connection originated from   |
-+-----------------+------------------------------------------------------+
-| error           | If login to destination failed, contains the error.  |
-+-----------------+------------------------------------------------------+
++---------------------+------------------------------------------------------+
+| Field               | Description                                          |
++=====================+======================================================+
+| source_port         | Source port where proxy connection originated from   |
```

Comment:
dot missing.

Reply:
This commit wasn't supposed to be here.. But will fix it in https://github.com/dovecot/documentation/pull/351